ec.pem

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEDTjFpi9+tJ8KnTPQff34Gpj4E4uc
+T4ML17IzkcWZhtY1O44fqMHmejS0umai8xJlD0Z/THhTiuRO8PC3vpkU9g==
+-----END PUBLIC KEY-----
```

Comment:
This one and `a0-signed-nitro.mender` files should be in a subdirectory inside `tests/`, ideally with an small README on how they have been generated.

Reply:
none of the other artifacts in the smoke tests are there. one day we must make order here. what I did last time I was here was to introduce the tests folder and the scripting around the tests. I moved the key and the artifact to the test/data directory and I added a read me.

Reply:
> none of the other artifacts in the smoke tests are there. 

What do you mean? Which other Artifacts? As far as I can see this is the only checked in artifact in the repo :mag_right: 

> (...), I moved the key and the artifact to the test/data directory and I added a read me

Thanks

Reply:
artifacts: ignore me it was a complicated situation, you are right. and it would take me so long to properly explain myself. my mistake in general.